ipam/claim.go

```diff
@@ -55,7 +61,9 @@ func (c *claim) Try(alloc *Allocator) bool {
 			alloc.infof("Claim %s for %s: address allocator still initializing; will try later.", c.cidr, c.ident)
 			c.sendResult(nil)
 		} else {
-			c.sendResult(fmt.Errorf("%s is in the range %s, but the allocator is not initialized yet", c.cidr, alloc.universe.AsCIDRString()))
+			alloc.infof(
+				"%s is in the range %s, but the allocator is not initialized yet",
+				c.cidr, alloc.universe.AsCIDRString())
```

Comment:
cosmetic: don't split this over multiple lines

also, the message should say "Claim %s for %s: is in the range %s, but ..."